### ISSUE TITLE
[BUG] cannot read property 'length' of undefined while doing a simple command

### ISSUE THREAD

**Issue by thefordyking** [npm/cli#2696]
I'm still getting this error in version 7.5.3 when I try to install a package and this was a big that was I think was fixed in this issue: https://github.com/npm/cli/pull/173

Here's the error I'm getting:
root@localhost:~/Desktop/fordycoin-miner# npm install node-fetch
npm ERR! Cannot read property 'length' of undefined

npm ERR! A complete log of this run can be found in:
npm ERR! /root/.npm/_logs/2021-02-14T04_59_20_565Z-debug.log

**Comment by nlf** [comment]
we need a lot more additional information before we can triage this with you. please open a new issue and be sure to fill out the full issue template, including what versions of node and npm you're using, and what steps we can take to reproduce this error.
